cmd/harmony/main.go

```diff
@@ -102,6 +103,9 @@ var (
 	shardID            = flag.Int("shard_id", -1, "the shard ID of this node")
 	enableMemProfiling = flag.Bool("enableMemProfiling", false, "Enable memsize logging.")
 	enableGC           = flag.Bool("enableGC", true, "Enable calling garbage collector manually .")
+	blsPrivateKey      = flag.String("blskey", "", "Bls serialized private key.")
```

Comment:
remove this support, this is too risky to have command line private key without passphrase.